src/libraries/System.IO/tests/Stream/Stream.NullTests.cs

```diff
@@ -131,16 +131,57 @@ public static void TestNullTextReader(TextReader input)
 
             if (sr != null)
                 Assert.True(sr.EndOfStream, "EndOfStream property didn't return true");
-            input.ReadLine();
+            Assert.Null(input.ReadLine());
             input.Dispose();
 
             input.ReadLine();
             if (sr != null)
                 Assert.True(sr.EndOfStream, "EndOfStream property didn't return true");
-            input.Read();
-            input.Peek();
-            input.Read(new char[2], 0, 2);
-            input.ReadToEnd();
+            Assert.Equal(-1, input.Read());
+            Assert.Equal(-1, input.Peek());
+
+            var chars = new char[2];
+            Assert.Equal(0, input.Read(chars, 0, chars.Length));
+            Assert.Equal(0, input.Read(chars.AsSpan()));
+            Assert.Equal(0, input.ReadBlock(chars, 0, chars.Length));
+            Assert.Equal(0, input.ReadBlock(chars.AsSpan()));
+            Assert.Equal("", input.ReadToEnd());
+            input.Dispose();
```

Comment:
the input has been already disposed. In case you want to test whether `Dispose()` can be called multiple times please add a separate test for that (it would be more obvious).
```suggestion
```